build.sbt

```diff
@@ -80,7 +80,6 @@ lazy val microservice = Project(appName, file("."))
       "-Xlint:unsound-match", // Pattern match may not be typesafe.
       "-Yno-adapted-args", // Do not adapt an argument list (either by inserting () or creating a tuple) to match the receiver.
       "-Ypartial-unification", // Enable partial unification in type constructor inference
-      "-Ywarn-dead-code", // Warn when dead code is identified.
```

Comment:
wasn't it possible to delete dead code instead?

Reply:
I get this error in my tests when mocking the authorise call => [error] /home/temp/Applications/hmrc-development-environment/hmrc/secure-message/test/uk/gov/hmrc/securemessage/controllers/SecureMessageControllerSpec.scala:133:43: dead code following this construct
[error]       when(mockAuthConnector.authorise(any(), any[Retrieval[Enrolments]])(any(), any()))